projects/pwa/src/component/elements/Text.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { css } from 'emotion';
+import { palette } from '@guardian/pasteup/palette';
+import { body } from '@guardian/pasteup/typography';
+
+// tslint:disable:react-no-dangerous-html
+export const textStyle = css`
+    strong {
+        font-weight: 700;
+    }
+    p {
+        padding: 0 0 12px;
+        ${body(2)};
+        font-weight: 300;
+        word-wrap: break-word;
+        color: ${palette.neutral[7]};
+    }
+    a {
+        color: ${palette.news.main};
+        text-decoration: none;
+        border-bottom: 1px solid ${palette.neutral[86]};
+        :hover {
+            border-bottom: 1px solid ${palette.news.main};
```

Comment:
could we use modern css underline styling?

Reply:
We don't use `text-decoration` in dotcom because we want the underline to be a different colour to the text. We can probably revisit the design with @zeftilldeath when we make this super pretty 😍